resource/META-INF/plugin.xml

```diff
@@ -167,5 +167,5 @@
 
   </actions>
 
-  <product-descriptor code="PZENUML" release-date="20190601" release-version="20191" optional="true" />
+  <product-descriptor code="PZENUML" release-date="20241230" release-version="20251" optional="true" />
```

Comment:
`verifyPlugin` requires the `release-version` must have the same beginning with `pluginVersion`. (`202412` and `2024.12.xx` don't work, looks a potential bug in the Gradle plugin)